mage_ai/frontend/components/datasets/FeatureProfiles.tsx

```diff
@@ -88,95 +115,73 @@ function FeatureProfile({
   const medianValue = statistics?.[`${uuid}/median`];
   const modeValue = statistics?.[`${uuid}/mode`];
   const numberOfInvalidValues = statistics?.[`${uuid}/invalid_value_count`];
+  const numberOfOutliers = statistics?.[`${uuid}/outlier_count`];
+  const skewness = statistics?.[`${uuid}/skew`];
+  const stddev = statistics?.[`${uuid}/std`];
 
-  let entries = [];
-  
-  if (COLUMN_TYPE_NUMBERICAL_WITH_DATETIME_LIKE.includes(columnType)) {
-    entries = [
-      [
-        'Type',
-        'Count',
-        'Unique',
-        'Missing',
-      ],
-      [
-        COLUMN_TYPE_HUMAN_READABLE_MAPPING[columnType] || columnType,
-        numberOfValues,
-        numberOfUniqueValues,
-        numberOfNullValues,
-      ],
-      [
-        columnType === ColumnTypeEnum.DATETIME ? 'Median' : 'Mean',
-        'Minimum',
-        'Maximum',
-        'Invalid',
-      ],
-      [
-        columnType === ColumnTypeEnum.DATETIME ? medianValue : roundNumber(meanValue, 3),
-        typeof minValue === 'string' ? minValue: roundNumber(minValue, 3),
-        typeof maxValue === 'string' ? maxValue: roundNumber(maxValue, 3),
-        numberOfInvalidValues,
-      ],
-    ];
-  } else {
-    entries = [
-      [
-        'Type',
-        'Count',
-        'Unique',
-      ],
-      [
-        COLUMN_TYPE_HUMAN_READABLE_MAPPING[columnType] || columnType,
-        numberOfValues,
-        numberOfUniqueValues,
-      ],
-      [
-        'Missing',
-        'Mode',
-        'Invalid',
-      ],
-      [
-        numberOfNullValues,
-        modeValue,
-        numberOfInvalidValues,
-      ],
-    ];
-  }
+  const entries = [
+    columnType,
+    numberOfUniqueValues,
+    meanValue,
+    medianValue,
+    modeValue,
+    minValue,
+    maxValue,
+    numberOfNullValues,
+    numberOfInvalidValues,
+    numberOfOutliers,
+    skewness,
+    stddev,
+  ];
 
   const featureSetId = featureSet.id;
   const featureUuid = feature?.uuid;
-  const featureId = getFeatureIdMapping(featureSet)[featureUuid]
+  const featureId = getFeatureIdMapping(featureSet)[featureUuid];
 
   return (
-    <FlexContainer>
-      <Flex flex={1}>
-        <CellStyle>
+    <Flex flexDirection="column">
+      <FeatureProfileStyle>
+        <Spacing p={2}>
           <NextLink
             as={`/datasets/${featureSetId}/features/${featureId}`}
             href="/datasets/[...slug]"
             key={featureUuid}
             passHref
           >
             <Link inline>
-              <Text backgroundColor={PURPLE_HIGHLIGHT} bold color={PURPLE} monospace>
+              <Text
+                backgroundColor={PURPLE_HIGHLIGHT}
+                bold
+                color={PURPLE}
```

Comment:
Instead of manually setting the color, use the Text prop that uses the color from the theme. The purple color might he "secondary" or add a prop to Text if needed.